libevmasm/CommonSubexpressionEliminator.h

```diff
@@ -130,7 +130,7 @@ class CSECodeGenerator
 	void generateClassElement(Id _c, bool _allowSequenced = false);
 	/// @returns the position of the representative of the given id on the stack.
 	/// @note throws an exception if it is not on the stack.
-	int classElementPosition(Id _id) const;
+	[[nodiscard]] int classElementPosition(Id _id) const;
```

Comment:
What is the idea behind marking this function nodiscard?

Reply:
the idea is just that if a function returns something, it need to be consumed. i'm not sure whether it makes sense to do this everywhere - probably not.